Sources/Parsing/Builders/OneOfBuilder.swift

```diff
@@ -18,6 +18,20 @@
 /// ```
 @resultBuilder
 public enum OneOfBuilder {
+  /// Provides support for specifying a parser in ``OneOfBuilder`` blocks.
+  @inlinable
+  public static func buildExpression<P: Parser>(_ parser: P) -> P {
+    parser
+  }
+
+  /// Provides support for specifying a substring parser in a UTF-8 ``OneOfBuilder`` block.
+  @inlinable
+  public static func buildExpression<P>(
+    _ parser: P
+  ) -> FromSubstring<Substring.UTF8View, P> {
```

Comment:
If this PR lands, we should probably also have a `buildExpression` for unicode scalars.